packages/core/src/baseclient.ts

```diff
@@ -371,6 +371,12 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
               return;
             }
 
+            if ('stacktrace' in finalEvent) {
```

Comment:
I would remove this, just bloats up the bundle size.
If we deprecate the property in TS in the type it's enough.